src/cuga/backend/loops/tools.py

```
@@ -0,0 +1,166 @@
+    await svc.register_loop(loop)
+    return f"Wakeup scheduled in {seconds}s — loop id {loop.id}."
+
+
+@tool
+async def list_my_loops() -> str:
+    """List all loops you (this agent + thread) currently have scheduled.
+
+    Returns a JSON array describing each loop's id, status, cadence, prompt,
+    and next fire time.
+    """
+    agent, thread, _ = _identity()
```

Ruff (0.15.12)

[warning] Unpacked variable `agent` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)